tests/integration/targets/ntnx_clusters_profiles_v2/tasks/clusters_profiles.yml

```
@@ -0,0 +1,1797 @@
+          should_inform: "{{ cluster_profiles[0].snmp_config.traps[0].should_inform }}"
+          engine_id: "{{ cluster_profiles[0].snmp_config.traps[0].engine_id }}"
+          version: "{{ cluster_profiles[0].snmp_config.traps[0].version }}"
+          reciever_name: "{{ cluster_profiles[0].snmp_config.traps[0].reciever_name }}"
+          community_string: "{{ cluster_profiles[0].snmp_config.traps[0].community_string }}"
+    rsyslog_server_list:
+      - server_name: "{{ cluster_profiles[0].rsyslog_server_list[0].server_name }}"
+        ip_address:
+          ipv4:
+            value: "{{ cluster_profiles[0].rsyslog_server_list[0].ip_address.ipv4.value }}"
+            prefix_length: "{{ cluster_profiles[0].rsyslog_server_list[0].ip_address.ipv4.prefix_length | int }}"
+        port: "{{ cluster_profiles[0].rsyslog_server_list[0].port | int }}"
+        network_protocol: "{{ cluster_profiles[0].rsyslog_server_list[0].network_protocol }}"
+        modules:
+          - name: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[0].name }}"
+            log_severity_level: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[0].log_severity_level }}"
+            should_log_monitor_files: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[0].should_log_monitor_files }}"
+          - name: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[1].name }}"
+            log_severity_level: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[1].log_severity_level }}"
+            should_log_monitor_files: "{{ cluster_profiles[0].rsyslog_server_list[0].modules[1].should_log_monitor_files }}"
+    pulse_status:
+      is_enabled: "{{ cluster_profiles[0].pulse_status.is_enabled }}"
+      pii_scrubbing_level: "{{ cluster_profiles[0].pulse_status.pii_scrubbing_level }}"
+  register: result
+  ignore_errors: true
+
+- name: Update cluster profile with wrong external ID status
+  ansible.builtin.assert:
+    that:
+      - result.changed == false
+      - result.failed == true
+      - result.msg == "Api Exception raised while fetching cluster profile info using ext_id"
+      - result.response is defined
+      - result.response.data.error | length > 0
+    fail_msg: "Update cluster profile with wrong external ID did not pass as expected"
+    success_msg: "Update cluster profile with wrong external ID passed as expected"
+
+- name: Fetch cluster profile using external ID
+  nutanix.ncp.ntnx_clusters_profiles_info_v2:
+    ext_id: "{{ cluster_profile_ext_id_2 }}"
+  register: result
+  ignore_errors: true
+
+- name: Assert cluster profile fetched successfully
+  ansible.builtin.assert:
+    that:
+      - result.changed == false
+      - result.failed == false
+      - result.ext_id == "{{ cluster_profile_ext_id_2 }}"
+      - result.response is defined
+      - result.response.ext_id == "{{ cluster_profile_ext_id_2 }}"
+    fail_msg: "Fetch cluster profile using external ID did not pass as expected"
```

Please change the messages.
```suggestion
    fail_msg: "Fetch cluster profile using external ID failed"
```